base.Dockerfile

```diff
@@ -10,7 +14,18 @@ ENV LANG=C.UTF-8 \
   RUBY_DOWNLOAD_SHA256=${RUBY_DOWNLOAD_SHA256}
 
 # Install build dependencies
-RUN install_packages build-essential bison dpkg-dev libgdbm-dev ruby wget autoconf libssl-dev zlib1g-dev libreadline-dev
+RUN install_packages build-essential bison dpkg-dev libgdbm-dev ruby wget autoconf zlib1g-dev libreadline-dev checkinstall
+
+# Build OpenSSL 1.1.1 (until we move away from Ruby <3.1)
```

Comment:
nit: consider making this a TODO, since it's something that can/should be cleaned up once something else has happened. e.g.
```suggestion
# TODO: stop building OpenSSL once all apps are on Ruby 3.1+.
```